packages/web/src/dev/CopyToClipboard.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from 'react'
+import { StyleSheet, Text, View } from 'react-native'
+import CopyIcon from 'src/icons/CopyIcon'
+import Hoverable from 'src/shared/Hoverable'
+import { colors } from 'src/styles'
+import { copyToClipboad } from 'src/utils/utils'
+
+interface Props {
+  content: string
+}
+
+interface State {
+  copied: boolean
+  hover: boolean
+}
+
+class CopyToClipboard extends React.PureComponent<Props, State> {
+  state = {
+    copied: false,
+    hover: false,
+  }
+
+  hoverIn = () => this.setState({ hover: true })
+  hoverOut = () => this.setState({ hover: false })
+
+  copy(event) {
+    event.preventDefault()
+    event.stopPropagation()
+    if (!this.state.copied) {
+      copyToClipboad(this.props.content)
+      this.setState({ copied: true })
+      setTimeout(() => this.setState({ copied: false }), 2000)
+    }
+  }
+
+  render() {
+    const { copied, hover } = this.state
+    const copy = this.copy.bind(this)
```

Comment:
same here with  arrow function replacing bind

Reply:
In this case, is obvious that it can be done easily. But there is a good way to do it if I need to send a param?
(I'll change it)

Reply:
forgive me if im not following but 

if copy is defined on a Component like
```
class Component {

copy = (param) => {
 // stuff in funciton as normal
}
}

```

you can access `this.copy()` like normal